pkg/config/default-docker-compose.yml

```diff
@@ -228,6 +238,31 @@ services:
     {{- with .AdditionalContent }}{{ marshalContent . }}{{- end }}
   {{- end }}
 
+  {{- with .Services.vote }}
+
+  vote:
+    image: {{ .ContainerRegistry }}/openslides-vote:{{ .Tag }}
+    {{- if checkFlag $.DisableDependsOn }}{{ else }}
+    depends_on:
+      - datastore-reader
+      - datastore-writer
+      - auth
+      - redis
+    {{- end }}
+    environment:
+      << : *default-environment
+      MESSAGING: redis
+      AUTH: ticket
+    networks:
+      - datastore-reader
+      - datastore-writer
```

Comment:
@ostcar Does the vote service use the datastore-reader and/or the datastore-writer directly? Or does it get all of its information from the backend?

Please also check networks alltogether. Maybe "frontend" is missing (see e. g. autoupdate and icc)

Reply:
It accesses the datastore-reader but not the datastore-writer.

It needs to be in the same network as the backend, the datastore-reader, the message bus, the Auth service and the autoupdate service.